File: comandor/src/main/java/com/komandor/komandor/widgets/MarginItemDecoration.java
package com.komandor.komandor.widgets;

import android.graphics.Rect;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class MarginItemDecoration extends RecyclerView.ItemDecoration {
  
  private final int spaceHeight;
  
  public MarginItemDecoration(int spaceHeight) {
    this.spaceHeight = spaceHeight;
  }
  
  @Override
  public void getItemOffsets(@NonNull Rect outRect, View view, RecyclerView parent,
                             RecyclerView.State state) {
    if (parent.getChildAdapterPosition(view) == 0) {
      outRect.bottom = spaceHeight;
    }
    
    outRect.top = spaceHeight;
    outRect.left = spaceHeight;
    outRect.right = spaceHeight;
  }
  
}

File: comandor/src/main/java/com/komandor/komandor/ui/adapter/MarginItemDecoration.kt
package com.komandor.komandor.ui.adapter

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.RecyclerView.*

class MarginItemDecoration(private val spaceSize: Int) : ItemDecoration() {
    override fun getItemOffsets(
        outRect: Rect, view: View,
        parent: RecyclerView,
        state: State
    ) {
        with(outRect) {
            if (parent.getChildAdapterPosition(view) == 0) {
                top = spaceSize
            }
            left = spaceSize
            right = spaceSize
            bottom = spaceSize
        }
    }
}